### ISSUE TITLE
Influxdb-ruby async options

### ISSUE THREAD

**Issue by dilnavazk** [InfluxCommunity/influxdb-rails#29]
Setting async options in the initializer is not working.

```
config.async = {
      # number of points to write to the server at once
      max_post_points:    1000,
      # queue capacity
      max_queue_size:     10_000,
      # number of threads
      num_worker_threads: 3,
      # max. time (in seconds) a thread sleeps before
      # checking if there are new jobs in the queue
      sleep_interval: 1000
    }
```

After setting this, there is no change in the workers sleep time. Is this the right setup? Or is this an issue with the library?

**Comment by dmke** [comment]
influxdb-rails depends on influxdb-ruby version 0.2.x, but the async options became available with 0.3.0.

@toddboom Would you add me to the maintainer list here, too?

**Comment by dilnavazk** [comment]
The issue is with the dependency over influxdb-ruby version 0.2.x. Solved by updating it to current version. Thanks @dmke
